wordle-web/app.vue

```diff
@@ -3,44 +3,79 @@
     <v-app>
       <v-app-bar color="primary" :elevation="2">
         <template v-slot:prepend>
-          <v-icon color="secondary"> mdi-book </v-icon>
+          <v-btn icon @click="router.push('/')">
+            <v-icon color="#FFFFFF">mdi-wizard-hat</v-icon> <!-- Wizard hat icon -->
```

Comment:
The white theme hides the logo, so I would make the color dependent on the theme, like color=secondary

Reply:
good call I just changed it to purple.